src/go/rdctl/pkg/plist/plist.go

```diff
@@ -73,15 +100,14 @@ func convertToPListLines(v reflect.Value, indent string) ([]string, error) {
 		retLines[numValues+1] = indent + "</array>"
 		return retLines, nil
 	case reflect.Map:
-		numValues := len(v.MapKeys())
-		if numValues == 0 {
-			return nil, nil
-		}
 		returnedLines := []string{indent + "<dict>"}
-		typedKeys := utils.SortKeys(v.MapKeys())
-		for _, typedKey := range typedKeys {
-			keyAsString := typedKey.StringKey
-			innerLines, err := convertToPListLines(v.MapIndex(typedKey.MapKey), indent+indentChange)
+		actualKeys := utils.SortKeys(value.MapKeys())
+		for _, actualKey := range actualKeys {
+			keyAsString := actualKey.StringKey
+			// If it's a map (always of string => bool|string|int), the typed and actual values are the same
+			// The only difference is that if the field isn't specified in the input, there will be an instance
+			// in `typedValue` but not `actualValue`.
```

Comment:
Comment needs updating — `typedValue` is no longer a thing (and `actualValue` got renamed).

This probably means `actualKeys` needs a new name too.

Reply:
No comment needed after renaming `actualKey*` to `mapKey*`